Packet++/header/ProtocolType.h

```diff
@@ -322,6 +322,11 @@ namespace pcpp
 	 */
 	const ProtocolType COTP = 0x900000000000;
 
+	/**
+	 * SLL2 protocol
+	 */
+	const ProtocolType SLL2 = 0x4000000000000;
```

Comment:
Can you please fix the protocol types?

```cpp
	/**
	 * COTP protocol
	 */
	const ProtocolType COTP = 0x4000000000000;

	/**
	 * SLL2 protocol
	 */
	const ProtocolType SLL2 = 0x8000000000000;
```

Reply:
Fixed here: https://github.com/seladb/PcapPlusPlus/pull/1138/commits/2d38623dfa5be2277d6038978dcef027cc938716